kirby/api/queue.py

```diff
@@ -3,13 +3,13 @@
 from smart_getenv import getenv
 import logging
 
-import datetime
+from .ext import Topic
 
 logger = logging.getLogger(__name__)
 
 
-class Queue:
-    def __init__(self, name, testing=False):
+class Queue(Topic):
+    def __init__(self, name, testing=False, security_protocol="SSL"):
```

Comment:
is adding the security protocol really needed here?

Reply:
It is now a boolean, if ssl_security_protocol is false, use plaintext